### ISSUE TITLE
loss never decrease

### ISSUE THREAD

**Issue by Bryce1010** [Fengdalu/Lipreading-DenseNet3D#6]
Hi, thanks for your work.  
I trained your network from scratch, but the loss was difficult to drop down.  
Did you cover that situation?

**Comment by Fengdalu** [comment]
> Hi, thanks for your work.
> I trained your network from scratch, but the loss was difficult to drop down.
> Did you cover that situation?

LRW pretrain is needed, described in our paper: https://arxiv.org/abs/1810.06990
